README.md

```diff
@@ -35,7 +35,8 @@ To start the services locally you will first need to create a `.env` file next t
         EMAIL_HOST_USER=SMTP_Injection
         EMAIL_PORT=587
         EMAIL_HOST_PASSWORD=<Api password need to retrieve from sparkpost>
-        ERROR_EMAIL=<email to recieve reports>   
+        ERROR_EMAIL=<email to recieve reports>
```

Comment:
Could I be picky and we use the `EMAIL` prefix for all of the email related variables with something like
```
EMAIL_FROM_ADDRESS=
EMAIL_TO_ADDRESS=
```
for the last 2 variables?